glossary.md

```diff
@@ -44,35 +56,105 @@ With the cached table feature, TiDB loads the data of an entire table into the m
 
 Coalesce Partition is a way of decreasing the number of partitions in a Hash or Key partitioned table. For more information, see [Manage Hash and Key partitions](/partitioned-table.md#manage-hash-and-key-partitions).
 
+### Column Family (CF)
+
+In RocksDB and TiKV, a Column Family (CF) represents a logical grouping of key-value pairs within a database.
+
+### Common Table Expression (CTE)
+
+A Common Table Expression (CTE) enables you to define a temporary result set that can be referred to multiple times within a SQL statement using the [`WITH`](/sql-statements/sql-statement-with.md) clause. For more information, see [Common Table Expression](/develop/dev-guide-use-common-table-expression.md).
+
 ### Continuous Profiling
 
 Introduced in TiDB 5.3.0, Continuous Profiling is a way to observe resource overhead at the system call level. With the support of Continuous Profiling, TiDB provides performance insight as clear as directly looking into the database source code, and helps R&D and operation and maintenance personnel to locate the root cause of performance problems using a flame graph. For details, see [TiDB Dashboard Instance Profiling - Continuous Profiling](/dashboard/continuous-profiling.md).
 
 ## D
 
+### Data Definition Language (DDL)
+
+Data Definition Language (DDL) is a part of the SQL standard that deals with creating, modifying, and dropping tables and other objects. For more information, see [DDL Introduction](/ddl-introduction.md).
+
+### Data Migration (DM)
+
+Data Migration (DM) is a tool for migrating data from MySQL-compatible databases into TiDB. DM reads data from a MySQL-compatible database instance and applies it to a TiDB target instance. For more information, see [DM Overview](/dm/dm-overview.md).
+
+### Data Modification Language (DML)
+
+Data Modification Language (DML) is a part of the SQL standard that deals with inserting, updating, and dropping rows in tables.
+
+### Development Milestone Release (DMR)
+
+Development Milestone Releases (DMR) are TiDB releases that introduce the latest features but do not offer long-term support. For more information, see [TiDB Versioning](/releases/versioning.md).
+
+### Disaster Recovery (DR)
+
+Disaster Recovery (DR) includes solutions that can be used to recover data and services from a disaster in the future. TiDB offers various Disaster Recovery solutions such as backups and replication to standby clusters. For more information, see [Overview of TiDB Disaster Recovery Solutions](/dr-solution-introduction.md).
+
+### Distributed eXecution Framework (DXF)
+
+Distributed eXecution Framework (DXF) is the framework used by TiDB to distribute tasks across a TiDB cluster. DXF is designed to efficiently use cluster resources to execute tasks (such as index creation or data import) while controlling the resource usage and impact on core business transactions. For more information, see [DXF Introduction](/tidb-distributed-execution-framework.md).
```

Comment:
```suggestion
Distributed eXecution Framework (DXF) is the framework used by TiDB to centrally schedule and execute certain tasks (such as creating indexes or importing data) in a distributed manner. DXF is designed to efficiently use cluster resources while controlling resource usage and reducing the impact on core business transactions. For more information, see [DXF Introduction](/tidb-distributed-execution-framework.md).
```

Reply:
The tasks are centrally scheduled, but not centrally executed. Maybe something like "centrally schedule ... and execute in distributed manner"?

Reply:
Done. PTAL again, thanks.